dev-python/confuse/confuse-1.0.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+inherit distutils-r1
+
+DESCRIPTION="Confuse is a configuration library for Python that uses YAML"
+HOMEPAGE="https://github.com/beetbox/confuse"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
```

Comment:
I see Github release includes tests. Could you switch to that if pypi tarball don't have them? And implement those python tests in this ebuild.

Reply:
Ah, yes, definitely :)